zebra-rpc/src/methods/get_block_template_rpcs/get_block_template.rs

```diff
@@ -164,6 +164,13 @@ where
     Tip: ChainTip + Clone + Send + Sync + 'static,
     SyncStatus: ChainSyncStatus + Clone + Send + Sync + 'static,
 {
+    // TODO:
+    // - Add a `disable_peers` field to `Network` to check instead of `disable_pow()` (#8361)
+    // - Check the field in `sync_status` so it applies to the mempool as well.
+    if network.disable_pow() {
+        return Ok(());
```

Comment:
###### Optional Suggestion

Let's mention in the function docs that it returns an `Ok` when PoW is disabled.

Reply:
Added to the function docs in https://github.com/ZcashFoundation/zebra/pull/8694/commits/3a730b7fcaba623179703daa6797cb886e6dc381.